pkg/command/context.go

```diff
@@ -1603,14 +1639,17 @@ func completionFormatCtxs(ctxs []*configtypes.Context) []string {
 	return comps
 }
 
-// dynamicTableWriter writes the data in table format dynamically by hiding column if all related rows for a column is `n/a`
-func dynamicTableWriter(slices interface{}, tableWriter component.OutputWriter) {
+// dynamicTableWriter writes the data in table format dynamically by
+// - showing all required columns
+// - optionally showing the dynamic columns if the data exists
+func dynamicTableWriter(slices interface{}, tableWriter component.OutputWriter, requiredColumns, dynamicColumns []string) {
```

Comment:
nit (preexisting): function name feels weird for what it does. How about renderDynamicTable?